src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionServiceImpl.java

```diff
@@ -49,6 +49,8 @@ public class SubscriptionServiceImpl implements SubscriptionService {
 
     @Override
     public Subscription createSubscription(Subscription subscription) {
+        duplicateSubscriptionHandler(subscription);
```

Comment:
Just needs some unit tests adding for this

Reply:
Done :)